app/views/publish/courses/subjects/_primary_form.html.erb

```diff
@@ -9,8 +9,8 @@
       </h1>
     </legend>
     <%= render "publish/shared/error_messages", error_keys: [:subjects] %>
-    <% primary_form_options.each do |opt| %>
-      <%= form.govuk_radio_button :master_subject_id, opt.code, label: { text: opt.name } %>
+    <% PrimarySubject.order(:subject_name).each do |subject| %>
```

Comment:
https://github.com/DFE-Digital/publish-teacher-training/pull/3315#discussion_r1093332871

Reply:
Is there a reason we need `PrimarySubject.includes(:financial_incentive).all` vs `PrimarySubject.order(:subject_name)` ?

Reply:
I don't think there is any financial incentives this recruitment cycle